_posts/2020-12-11-coordinate-systems.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "Support for alternate input and output coordinate systems"
+date:   2020-12-11 17:00:00
+author: Rich Gwozdz
+---
+
+With the recent release of Winnow 2.2.0, the Koop Geoservices (FeatureServer) output-plugin now supports alternate coordinate reference systems (CRSs) for input and output data. Specifically:
```

Comment:
It is better to specify the geoservices or Koop version

Reply:
Ok, no problem.  I'll need to bump it's version in geoservices first then.

Reply:
I switch the reference to the top level dependency, Geoservices.